antarest/storage/business/study_service.py

```diff
@@ -76,77 +75,70 @@ def get(self, route: str, depth: int) -> JSON:
         del study
         return data
 
-    def get_study_information(self, uuid: str) -> JSON:
-        config = StudyConfig(study_path=self.path_to_studies / uuid)
+    def get_study_information(self, metadata: Metadata) -> JSON:
+        config = StudyConfig(study_path=self.get_study_path(metadata))
         study = self.study_factory.create_from_config(config)
         return study.get(url=["study"])
 
-    def get_studies_information(self) -> JSON:
-        return {
-            uuid: self.get_study_information(uuid)
-            for uuid in self.get_study_uuids()
-        }
+    def get_workspace_path(self, workspace: str) -> Path:
+        return Path(self.config[f"storage.workspaces.{workspace}.path"])
 
-    def get_study_path(self, uuid: str) -> Path:
-        return self.path_to_studies / uuid
+    def get_study_path(self, metadata: Metadata) -> Path:
+        path: Path = self.get_workspace_path(metadata.workspace) / metadata.id
+        return path
 
-    def create_study(self, study_name: str) -> str:
+    def create_study(self, metadata: Metadata) -> Metadata:
         empty_study_zip = self.path_resources / "empty-study.zip"
 
-        uuid = StorageServiceUtils.generate_uuid()
-
-        path_study = self.get_study_path(uuid)
+        path_study = self.get_study_path(metadata)
         path_study.mkdir()
 
         with ZipFile(empty_study_zip) as zip_output:
             zip_output.extractall(path=path_study)
 
-        study_data = self.get(uuid, 10)
-        StorageServiceUtils.update_antares_info(study_name, study_data)
+        study_data = self.get(metadata, metadata.id, 10)
```

Comment:
second arg should be the data route, so here the root "".
also the depth should be -1 since we want the whole study data to save